lib/validator/v3/validator.go

```diff
@@ -1234,6 +1235,22 @@ func validateRuleMetadata(structLevel validator.StructLevel) {
 	validateObjectMetaAnnotations(structLevel, ruleMeta.Annotations)
 }
 
+func validateRouteTableRange(structLevel validator.StructLevel) {
+	r := structLevel.Current().Interface().(api.RouteTableRange)
+	if r.Min >= 1 && r.Max >= r.Min && r.Max <= 250 {
```

Comment:
I noticed [1-252] as valid table id values. Did we deliberately leave out 251,252?

Thanks